src/meta/app/src/principal/user_stage.rs

```diff
@@ -561,7 +561,7 @@ pub struct StageInfo {
     pub stage_name: String,
     pub stage_type: StageType,
     pub stage_params: StageParams,
-    pub is_from_uri: bool,
+    pub is_temporary: bool,
```

Comment:
```suggestion
    // on `COPY INTO xx FROM 's3://xxx?ak=?&sk=?'`, the URL will be treated as an temporary stage.
    pub is_temporary: bool,
```

how about adding a line of comment about the semantics of this field?

Reply:
Done